src/components/Walk/ResultsPage.js

```diff
@@ -2,22 +2,120 @@ import React from "react";
 import Typography from '@mui/material/Typography';
 import Container from '@mui/material/Container';
 import Grid from '@mui/material/Grid';
+import QuestionGraph from "../Graphs/QuestionGraph";
+import WalkGraph from "../Graphs/WalkGraph";
+
+/**
+ * GRAPH 1: Questions Data format
+ */
+const questionAnswers = [
```

Comment:
1st Graph will contain data about questions and answers. It will contain how many people chose a specific answer.